catalog.yaml

```diff
@@ -1,36 +1,41 @@
-sources:
-  EU:
-    description: EU Node  
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/EU/main.yaml"
-  NERSC:
-    description: NERSC Node  
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/NERSC/main.yaml"
-  NCAR:
-    description: NCAR Node  
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/NCAR/main.yaml"
-  online:
-    description: online data sources
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/online/main.yaml"
-  UK:
-    description: online data sources
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/UK/main.yaml"
-  JAPAN:
-    description: online data sources
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/JAPAN/main.yaml"
-  CN:
-    description: earthlab data sources
-    driver: yaml_file_cat
-    args:
-        path: "{{CATALOG_DIR}}/CN/main.yaml"
+sources:
+  EU:
+    description: EU Node  
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/EU/main.yaml"
+  NERSC:
+    description: NERSC Node  
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/NERSC/main.yaml"
+  NCAR:
+    description: NCAR Node  
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/NCAR/main.yaml"
+  online:
+    description: online data sources
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/online/main.yaml"
+  UK:
+    description: online data sources
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/UK/main.yaml"
+  JAPAN:
+    description: online data sources
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/JAPAN/main.yaml"
+  CN:
+    description: earthlab data sources
+    driver: yaml_file_cat
+    args:
+        path: "{{CATALOG_DIR}}/CN/main.yaml"

```

Comment:
This change also changes all the lines. I would have expected a change which only adds the `SP` entry.